site/src/pages/TerminalPage/TerminalPage.tsx

```diff
@@ -38,6 +38,10 @@ const TerminalPage: FC<
     const search = new URLSearchParams(location.search)
     return search.get("reconnect") ?? uuidv4()
   })
+  const [command] = useState(() => {
+    const search = new URLSearchParams(location.search)
+    return search.get("command") || undefined
+  })
```

Comment:
I don't think this need to be a state since we don't change it.  You could make something like:
```
 const [searchParams] = useSearchParams()
 const command = searchParams.get("command")
```

Reply:
Ahh, neato!

Reply:
Hm... I think react-router-dom does something similar on their back but even in a loop, parsing the search parameters should not be an issue tho

Reply:
Ohhhh, I missed your sample. I'll update it to do that.

Reply:
Fixed!